packages/nql/test/integration/nql_mingo.test.js

```
@@ -413,4 +413,46 @@ describe('Integration with Mingo', function () {
             query.queryJSON(advancedJSON.posts[4]).should.eql(false);
         });
     });
+
+    describe.only('Dates', function () {
```

ESLint

[error] Unexpected exclusive mocha test.

(ghost/mocha/no-exclusive-tests)